autogluon/build_artifacts/inference/config.properties

```diff
@@ -2,4 +2,25 @@ vmargs=-XX:+UseContainerSupport -XX:InitialRAMPercentage=8.0 -XX:MaxRAMPercentag
 model_store=/opt/ml/model
 load_models=ALL
 inference_address=http://0.0.0.0:8080
-management_address=http://0.0.0.0:8081
\ No newline at end of file
+management_address=http://0.0.0.0:8081
+enable_envvars_config=true
+# management_address=unix:/tmp/management.sock
```

Comment:
Do we want to keep these?

Reply:
yes; for documentation purposes (it's the same config as in pytorch images)